config/kustomization.yaml

```diff
@@ -18,15 +18,15 @@ namePrefix: istio-operator-
 # YAML string, with resources separated by document
 # markers ("---").
 resources:
-- ../rbac/rbac_role.yaml
-- ../rbac/rbac_role_binding.yaml
-- ../manager/manager.yaml
+- rbac/rbac_role.yaml
+- rbac/rbac_role_binding.yaml
+- manager/manager.yaml
   # Comment the following 3 lines if you want to disable
   # the auth proxy (https://github.com/brancz/kube-rbac-proxy)
   # which protects your /metrics endpoint.
-- ../rbac/auth_proxy_service.yaml
-- ../rbac/auth_proxy_role.yaml
-- ../rbac/auth_proxy_role_binding.yaml
+- rbac/auth_proxy_service.yaml
+- rbac/auth_proxy_role.yaml
+- rbac/auth_proxy_role_binding.yaml
 
 patches:
 - manager_image_patch.yaml
```

Comment:
You have moved the `kustomization.yaml` file from `config/default` to `config` (along with `manager_auth_proxy_patch.yaml` and `manager_prometheus_metrics_patch.yaml`). There is however, this `manager_image_patch.yaml` file which should be located under `config/default` according to the [Makefile](https://github.com/banzaicloud/istio-operator/blob/3804b23719199ada4a7e8f78484c04186c9d8a08/Makefile#L91). (This file is added to [.gitignore](https://github.com/banzaicloud/istio-operator/blob/3804b23719199ada4a7e8f78484c04186c9d8a08/.gitignore#L32) so that it is not overwritten by developers who use different images for development.)

In this line, simply `manager_image_patch.yaml` is referenced from the `config` directory, but to align with the Makefile it should be searched under `config/default`.

```suggestion
- default/manager_image_patch.yaml
```

(Alternatively, this line could stay like this and the line in the [Makefile](https://github.com/banzaicloud/istio-operator/blob/3804b23719199ada4a7e8f78484c04186c9d8a08/Makefile#L91) could be changed to `./scripts/image_patch.sh ./config/manager_image_patch.yaml ${IMG}`, but I'd rather keep this file separately under `config/default`.)

Reply:
Good catch @Laci21 ,
I forgot to fix this one. Thanks for reviewing.

Reply:
Do you reckon I should move `manager_auth_proxy_patch.yaml` and `manager_prometheus_metrics_patch.yaml` back to `default` directory as well? 🤔 

I'm not really across the history behind this image_patching so I've asked question in slack...